<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HJNZL001711" failures="0" tests="1" name="com.FilpkartApplication.TestCases.TC_Verify_flipkartApplication" time="0.000" errors="1" timestamp="10 Dec 2021 06:59:27 GMT" skipped="0">
  <testcase classname="com.FilpkartApplication.TestCases.TC_Verify_flipkartApplication" name="Search_Mobiles" time="0.000">
    <error message="
Some DataProvider public static java.lang.Object[][] com.FilpkartApplication.utilities.ExcelDataProvider.requiredData(java.lang.String) throws java.io.IOException parameters unresolved:  at 0 type class java.lang.String
" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Some DataProvider public static java.lang.Object[][] com.FilpkartApplication.utilities.ExcelDataProvider.requiredData(java.lang.String) throws java.io.IOException parameters unresolved:  at 0 type class java.lang.String

at org.testng.internal.MethodInvocationHelper.getParameters(MethodInvocationHelper.java:197)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:130)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
    </error>
  </testcase> <!-- Search_Mobiles -->
</testsuite> <!-- com.FilpkartApplication.TestCases.TC_Verify_flipkartApplication -->
